proxy/core/src/graal/META-INF/native-image/com.typesafe/ssl-config/reflect-config.json.conf

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "name": "com.typesafe.sslconfig.ssl.NoopHostnameVerifier"
```

Comment:
@jroper @pvlugter We need to decide on what to choose for our usage—I have no idea and the documentation on the matter was not very helpful either. Right now it loads the NoopHostnameVerifier.

Reply:
Depends on what it's being used for.

Reply:
@patriknw Do you have any suggestion what what hostname verifier we should be using?

Reply:
This is sslconfig. That isn't used for Akka remoting. It was deprecated in Akka 2.6. I would be surprised if it's used by Akka gRPC. @raboof do you know? Or is it still used in Akka HTTP?